pkg/workloadapi/security/authorization.proto

```diff
@@ -34,6 +34,10 @@ message Authorization {
   // take place.
   // Groups are OR-ed.
   repeated Group groups = 5;
+
+  // Whether or not this is a dry run policy.
+  // Dry run policies are not enforced, but their matches are logged
+  bool dry_run = 6;
```

Comment:
I always flinch when an API has a boolean instead of a enum (though I did this in my PR too). Is there a world where there is some attribute that would be mutually exclusive with `dry_run`? I don't think so, but maybe someone else is more imaginative.